_includes/panel_action_modal.html

```diff
@@ -106,46 +105,39 @@
           '<option' + field_value + subtext + disabled + '>' + legislator.name + '</option>'
         );
       });
-      $('.btn-group.bootstrap-select').hide()
+      $('.btn-group.bootstrap-select').remove()
       $nomineeSelect.removeClass('hidden').selectpicker();
     }
   });
 }
 
-
 $(document).ready(function(){
   fetch_legislator_info();
 
-  $(".show-nomination-modal").click(function() {
-    $("#js-nomination-form").trigger("reset");
+  $(".show-nomination-modal").on("click", function() {
+    $("textarea, select").val(function() {
+        return this.defaultValue;
+    });
     $("#additional-fields").show();
-    $("#nomination-page-2").hide();
+    $("#nomination-page-thanks").hide();
     $("#nomination-page-1").show();
     $("#submit-candidate-modal").modal("show");
     $(".bootstrap-select .filter-option").text("Search by name or state");
   });
 
-  $("#submit-again").click(function() {
-    $("#nomination-page-2").hide();
+  $("#submit-again").on("click", function() {
+    $("#nomination-page-thanks").hide();
     $("#nomination-page-1").show();
   });
 
   $("#js-nomination-form").submit(function (event) {
     event.preventDefault();
-
-    if ($("select[name=legislator_id]").val() == "" && $("input[name=email]").val() == "") {
-      $("button.selectpicker").focus();
-    } else {
-      var saveURI = '{{site.services_url}}/nominations';
-      $.post(saveURI, $(this).serialize());
-      $("#nomination-page-1").hide();
-      $("#nomination-page-2").show();
-      $("#additional-fields").hide();
-      $("textarea[name=selection_comment]").val("");
-      $("textarea[name=other_comment]").val("");
-      $("select[name=legislator_id]").val("")
```

Comment:
Yes. This prevents us from getting empty email submission on subsequent
nominations. I'm resetting some fields on line 119, but don't want to reset the whole form.